src/mudlet.cpp

```diff
@@ -1257,7 +1257,19 @@ bool mudlet::setLabelClickCallback( Host * pHost, const QString & name, const QS
     QMap<QString, TLabel *> & labelMap = mHostLabelMap[pHost];
     if( labelMap.contains( name ) )
     {
-        labelMap[name]->setScript( pHost, func, pA );
+        labelMap[name]->setClick( pHost, func, pA );
```

Comment:
I would also have to modify the setClick function itself to have it return something to get this working, yeah? 
The current setClick is a void, so I'm guessing returning it without modifying the code would just have it returning NULL?
void TLabel::setClick( Host * pHost, const QString & func, const TEvent & args )
{
    mpHost = pHost;
    mClick = func;
    mClickParams = args;
}